.github/workflows/test.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: Test
+on: [push, pull_request]
+jobs:
+  molecule-clone:
+    name: Molecule Clone
+    runs-on: ubuntu-latest
+    container: quay.io/ansible/toolset:latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Molecule Test scenario
+        run: molecule test -s filter-plugin-clone
+  molecule-collection:
+    name: Molecule Collection
+    runs-on: ubuntu-latest
+    container: quay.io/ansible/toolset:latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          path: ansible_collections/confluent/platform
+      - name: Molecule Test scenario
+        run: molecule test -s filter-plugin-collection
+        working-directory: ansible_collections/confluent/platform
```

Comment:
I'm not sure i see how these two tests are different. one clones into current directory and then runs molecule. the other clones into `ansible_collections/confluent/platform` and then runs molecule from there. I do think we want to test both modes though. Let me read thru that blog post

Reply:
That is the trick used to test the collection behavior, and it seems to be adopted practice to test collections this way. When you use the standard Ansible Collection download/clone location for your collection as working directory, Ansible will find the collection at test already installed, and be able to use it - as any other collection user. Since the collection is the system at test, you would have to use the Git coordinates to make this work otherwise. This is just so simple - select your working directory. Try change it, and see what happens! 😄

Reply:
Outdated - change removed from PR